Resources/Prototypes/SimpleStation14/Species/shadowkin.yml

```diff
@@ -0,0 +1,161 @@
+﻿- type: species
+  id: Shadowkin
+  name: species-name-shadowkin
+  roundStart: true
+  prototype: MobShadowkin
+  sprites: MobShadowkinSprites
+  defaultSkinTone: "#FFFFFF"
```

Comment:
Is there a default eye color or something similar? Would be good to avoid clueless people never getting powers.

Reply:
Laki asked me about why their Shadowkin never got any powers and I realized that taking away their powers for simply having black eyes is pretty lame and is not communicated, I'll probably remove that and keep it a trait

Reply:
Would you like to do this on the next update or should I remove the eye color blackeye-ing here myself?